packages/core/src/components/tag/tag.stories.js

```diff
@@ -33,3 +39,27 @@ export const LargeDeletable = () => `
     </button>
   </div>
 `;
+
+export const LongText = () => `
+  <div class="hds-tag" style="max-width: 300px;">
+    <span class="hds-tag__label">Label - This is a tag with a very long text which is not advisable and might span into multiple lines</span>
+  </div>
+`;
+
+export const LongTextDeletable = () => `
+  <div class="hds-tag" style="max-width: 300px;">
+  <span class="hds-tag__label" aria-hidden="true">Label - This is a tag with a very long text which is not advisable and might span into multiple lines</span>
+    <button aria-label="Delete item: Label" class="hds-tag__delete-button button-reset">
+      <span aria-hidden="true" class="hds-icon hds-icon--cross"></span>
+    </button>
+  </div>
+`;
+
+export const LargeLongTextDeletable = () => `
```

Comment:
Core is missing LargeLongText story that React storybook has. Also the stories in React are in a better order as they are sort of grouped by the context, e.g. Large Tag stories are grouped, after that Long Text Tags (default). Could you put the stories in the same order in Core side as well?

Reply:
Will do

Reply:
Done